### DISCUSSION TITLE
Bookmark individual files to home page to easy viewing

### DISCUSSION THREAD

**Discussion by ariesfans** [community/community#5817] [Mobile]
some files need check but so many dirctories

**Comment by brianlovin** [comment]
Thanks for the suggestion @ariesfans — for my own curiosity, what's your use case for needing to get to specific files frequently? Are they reference files, or files that change frequently?

**Comment by ariesfans** [comment]
thanks for your reply&nbsp;
due to the following reasons&nbsp;
first our team project’s documents write with markdown&nbsp;
sec some repositories just use save technology notes&nbsp;









------------------ Original ------------------
From: Brian Lovin ***@***.***&gt;
Date: Tue,Nov 9,2021 4:06 AM
To: github/feedback ***@***.***&gt;
Cc: howsopen ***@***.***&gt;, Mention ***@***.***&gt;
Subject: Re: [github/feedback] Bookmark individual files to home page to easyviewing (Discussion #5817)





 
Thanks for the suggestion @ariesfans — for my own curiosity, what's your use case for needing to get to specific files frequently? Are they reference files, or files that change frequently?
